README.rst

```diff
@@ -13,9 +13,63 @@ It also handles file finders for the supported SCMs.
 .. image:: https://tidelift.com/badges/github/pypa/setuptools_scm
    :target: https://tidelift.com/subscription/pkg/pypi-setuptools_scm?utm_source=pypi-setuptools_scm&utm_medium=readme
 
+``pyproject.toml`` usage
+------------------------
+
+The preferred way to configure ``setuptools_scm`` is to author
+settings in a ``tool.setuptools_scm`` section of ``pyproject.toml``.
+
+This feature requires Setuptools 42 or later, released in Nov, 2019.
+If your project needs to support build from sdist on older versions
+of Setuptools, you will need to also implement the ``setup.py usage``
```

Comment:
Looks like you placed the closing backticks too far...

Reply:
I wasn't quite sure how to handle that, given that the backticks are in the title. I'm happy to accept edits on it.

Reply:
Oh. I'm not sure either. Could probably try using a link syntax.